master/buildbot/test/unit/test_worker_ec2.py

```diff
@@ -138,7 +138,7 @@ def test_start_vpc_instance(self):
         bs = ec2.EC2LatentWorker('bot1', 'sekrit', 'm1.large',
                                  identifier='publickey',
                                  secret_identifier='privatekey',
-                                 keypair_name="test_key",
+                                 keypair_name="latent_buildbot_slave",
```

Comment:
I changed the value from `test` to `worker` in every test that wasn't checking for deprecation warnings.. there's quite a few that did that.